tests/automation.js

```diff
@@ -546,6 +546,31 @@ casper.test.begin("unit tests", 25 + gfxTests.length, function(test) {
         test.assertTextDoesntExist("FAIL");
     });
 
+    // The following test tests that a MIDlet can be started, paused and then started again.
+    casper
+    .thenOpen("http://localhost:8000/index.html?midletClassName=tests.background.BackgroundMIDlet1&jad=tests/midlets/background/pause.jad&jars=tests/tests.jar&logConsole=web,page&logLevel=log")
+    .withFrame(0, function() {
+        casper.waitForText("startApp", function() {
+            casper.waitForText("pauseApp", function() {
+                casper.waitForText("startApp", function() {
+                    var content = this.getPageContent();
+                    test.assertEquals(content.match(/startApp/g).length, 2, "MIDlet started twice");
```

Comment:
IIRC I expressly tried with a single *startApp* and the test didn't pass, but I probably remember wrong.